src/ServiceManagement/StorSimple/Commands.StorSimple/Exceptions/NoDeviceRegisteredException.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Exceptions
+{
+    [Serializable]
+    public class NoDeviceRegisteredException : Exception
```

Comment:
You could have a static class that have static methods which return exception instances with the message you are looking for